code/game/objects/items/storage/belt.dm

```diff
@@ -664,6 +664,32 @@
 		/obj/item/melee/sabre
 		))
 
+/obj/item/storage/belt/sabre/rapier
+	name = "rapier sheath"
+	desc = "A black, thin sheath that looks to house only a long thin blade. Feels like its made of metal."

```

Comment:
And another painful description.

A sheath houses a single blade meaning "only" is entirely redundant. What else would be in there?